File: Aplicação/BuildingBug/kotlin-client/src/main/kotlin/io/swagger/client/models/Servico.kt
package io.swagger.client.models

import java.time.LocalDateTime


data class Servico (

    var id: Int,
    var morada: String,
    var dataInicio: LocalDateTime,
    var dataFim: LocalDateTime,
    var avaliacao: Int,
    var custo: Int,
    var fator: Int,
    var inactive: Boolean,
    var clienteId: Int,
    var estadoServicoId: Int,
    var codigoPostalCP: Int,
    var profissionalId: Int?,
    var tipoId: Int
)

File: Application Programming Interface/BuildingBug.Domain/Model/Servico.cs
using System.Text.Json.Serialization;

namespace BuildingBug.Domain.Model
{
    public class Servico
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }

        [JsonPropertyName("morada")]
        public string Morada { get; set; }

        [JsonPropertyName("dataInicio")]
        public DateTime DataInicio { get; set; }

        [JsonPropertyName("dataFim")]
        public DateTime DataFim { get; set; }

        [JsonPropertyName("avaliacao")]
        public int Avaliacao { get; set; }

        [JsonPropertyName("custo")]
        public float Custo { get; set; }

        [JsonPropertyName("fator")]
        public int Fator { get; set; }

        [JsonPropertyName("inactive")]
        public bool Inactive { get; set; }

        [JsonPropertyName("clienteId")]
        public int ClienteId { get; set; }

        [JsonPropertyName("estadoServicoId")]
        public int EstadoServicoId { get; set; }

        [JsonPropertyName("codigoPostalCP")]
        public int CodigoPostalCP { get; set; }

        [JsonPropertyName("profissionalId")]
        public int ProfissionalId { get; set; }

        [JsonPropertyName("tipoId")]
        public int TipoId { get; set; }
    }
}